### ISSUE TITLE
Orthographic issue Dameli [dml]

### ISSUE THREAD

**Issue by meesumalam** [common-voice/common-voice#4706]
Hi,

In dml, we are having orthographic issue, we submitted the words with the letter on the left top of the picture (which has four dots on it, whereas the cv display is showing as three dots above and three below.

![ima_220c53f](https://github.com/user-attachments/assets/a6924deb-239c-4f2a-8683-5f4eb63e0949)
how can we resolve this issue in dml?

Thanks

**Comment by KathyReid** [comment]
Hi @meesumalam, firstly a huge thank you for your contributions in the Dameli language, and apologies that this is occurring for your language.

I would like to confirm some details with you first. I apologise that I am not familiar with Arabic, Urdu or Persian orthography, so please forgive any mistakes. 

1. The character that is _expected_ is the character `ݜ` - Arabic sīn (seen) with four dots above, _however_, 

2. The character that is being _displayed_ on screen is the character `ڜ` which is Arabic sīn (seen) with three dots above and three dots below, which is sometimes used in Moroccan Arabic for loan words. 

My approach to diagnosing this issue is: 

1. Is the character _saved correctly_ when the sentence is added to Common Voice? 
2. If the character is _saved correctly_ to the database, then is there a _rendering_ issue when it displayed on screen? 

Would you please past the entire sentence so I can check in the Common Voice database? 

Many thanks in advance, 
Kathy

**Comment by HarikalarKutusu** [comment]
If it is because of the font, here is a similar issue:
https://github.com/common-voice/common-voice/issues/4678

**Comment by KathyReid** [comment]
Thanks @HarikalarKutusu - I had not considered some sort of font character substitution issue, that's a great suggestion to rule out.

**Comment by HarikalarKutusu** [comment]
I read some reports of problems for Open Sans font wrt RTL languages. We need to know the Unicode for problem characters. It might even be that there is no Unicode page defined for under represented languages. This would be huge problem for MCV's future. Not exactly an MCV issue, but we will pass 200 language limit in a couple of months, things can get rough.

@meesumalam, please pass a non-image character. If possible, in addition to that char, full alphabet will also be helpful.

**Comment by moz-kathyreid** [comment]
I agree @HarikalarKutusu - I think this is a broader issue with the solution being:

1. Needing 1 font to service 200+ languages and their orthographies
2. _or_ needing fonts to change on a per-language basis (which is more complex)

This is a _scaling_ issue - as CV supports more languages and more orthographies we are likely to run into glyphs that are not supported by one font, particularly as you say for under-represented languages. 

Have created a new Label `Font` to cluster these Issues.

**Comment by ftyers** [comment]
@meesumalam is this still a problem?

**Comment by meesumalam** [comment]
Nop,

We invalidated all the sentences with error.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Francis Tyers ***@***.***>
Sent: Thursday, June 19, 2025 8:57:59 AM
To: common-voice/common-voice ***@***.***>
Cc: Alam, Meesum ***@***.***>; Mention ***@***.***>
Subject: [External] Re: [common-voice/common-voice] Orthographic issue Dameli [dml] (Issue #4706)


This message was sent from a non-IU address. Please exercise caution when clicking links or opening attachments from external sources.


[https://avatars.githubusercontent.com/u/449545?s=20&v=4]ftyers left a comment (common-voice/common-voice#4706)<https://github.com/common-voice/common-voice/issues/4706#issuecomment-2988023107>

@meesumalam<https://github.com/meesumalam> is this still a problem?

**Comment by ftyers** [comment]
Thanks!
